src/mongodb.appup.src

```diff
@@ -1,27 +1,49 @@
 %% -*- mode: erlang -*-
-{"3.0.11",
+{"3.0.12",
  [
-  {"3.0.10", []},
+  {"3.0.11", [
+    {load_module, mc_topology, brutal_purge, soft_purge, []},
+    {load_module, mc_topology_logics, brutal_purge, soft_purge, []}
+  ]},
+  {"3.0.10", [
+    {load_module, mc_topology, brutal_purge, soft_purge, []},
+    {load_module, mc_topology_logics, brutal_purge, soft_purge, []}
+  ]},
   {"3.0.9", [
-    {load_module, mc_worker_logic, brutal_purge, soft_purge, []}
+    {load_module, mc_worker_logic, brutal_purge, soft_purge, []},
+    {load_module, mc_topology, brutal_purge, soft_purge, []},
+    {load_module, mc_topology_logics, brutal_purge, soft_purge, []}
   ]},
   {"3.0.8", [
     {load_module, mc_worker_logic, brutal_purge, soft_purge, []},
-    {load_module, mc_monitor, brutal_purge, soft_purge, []}
+    {load_module, mc_monitor, brutal_purge, soft_purge, []},
+    {load_module, mc_topology, brutal_purge, soft_purge, []},
+    {load_module, mc_topology_logics, brutal_purge, soft_purge, []}
   ]},
   {<<"3.0.[0-7]">>, [
     {restart_application, mongodb}
   ]},
   {<<".*">>, []}
  ],
  [
-  {"3.0.10", []},
+  {"3.0.11", [
```

Comment:
For downgrading from 3.0.12 -> 3.0.11, we need to reload all the 4 modules that are changed using `load_module`, and delete the new module mc_util using `{delete_module, mc_util}`.